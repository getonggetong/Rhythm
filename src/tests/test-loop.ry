init() {

	allNotes = [];
	start = A4;

	loop(0:12) {
		allNotes ::= start;
		start++;
	}

	print(allNotes);

}