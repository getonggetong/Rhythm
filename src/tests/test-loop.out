[A4,A#4.4,B4.4,C4.4,C#4.4,D4.4,D#4.4,E4.4,F4.4,F#4.4,G4.4,G#4.4]